packages/eddsa-poseidon/src/index.ts

```diff
@@ -1,2 +1 @@
-export * from "./eddsa-poseidon"
-export * from "./types"
```

Comment:
They haven't been removed, they're actually just being exported from the inner file to avoid duplication. Happy for any feedback on the export structure

Reply:
Oh yea, that should work 👍🏽